common/cpp/types.h

```diff
@@ -27,16 +27,6 @@
 #include <vector>
 #include <list>
 
-/** Avalon worker enclave Type (singleton, KME, WPE) */
```

Comment:
Why did you move this to tcf_error.h?
File types.h seems more appropriate as these are not error codes.

Reply:
I did move the enclave type enum to tcf_error.h because there are other enums in place. Also, types.h exposes lot of APIs like ByteArrayToString, ByteArrayToStringArray. This was resulting in undefined symbol errors during python module load time. Hence, i moved it to tcf_error.h

Reply:
Please create a new file to add this enum.